polaris-react/src/components/FullscreenBar/FullscreenBar.tsx

```diff
@@ -4,7 +4,6 @@ import {ExitMajor} from '@shopify/polaris-icons';
 import {Icon} from '../Icon';
```

Comment:
[This PR](https://5d559397bae39100201eedc1-tnpnnmnlft.chromatic.com/?path=/story/all-components-fullscreenbar--all)
[Production](https://storybook.polaris.shopify.com/?path=/story/all-components-fullscreenbar--all&globals=polarisSummerEditions2023:true)